index.js

```diff
@@ -179,10 +183,11 @@ function parseArch(opts) {
     opts.arch = 'i686';
   } else if (opts.bits === '32' && opts.platform === 'win32') {
     opts.arch = 'i386';
+  } else if (opts.bits === '64' && opts.platform === 'linux' && ARCH === 'arm64-ubuntu1604') {
```

Comment:
It would be great if you can make this package work with any AArch64 Linux distro. It looks like this change will only work with ubuntu1604. Can you make this check more general?

Reply:
Thanks for the review @rhenwood-arm 
The MongoDB server gets the download from location https://www.mongodb.com/download-center/community.
Currently, only **arm64-ubuntu1604** MongoDB server is present for arm user to download.

Reply:
thanks - that makes sense. Please add this comment into the commit so future readers of the code will understand why there is a specific test for ubuntu1604.